src/app/routes/utils/fetchDataFromBFF/index.ts

```diff
@@ -53,7 +53,8 @@ export default async ({
   });
 
   const agent = isLocal || BFF_IS_LOCAL ? undefined : await getAgent();
-  const timeout = isLocal && BFF_IS_LOCAL ? 60000 : null;
+  const timeout = 60000;
```

Comment:
Will this not affect the primary timeout? https://github.com/bbc/simorgh/blob/7d122f4ffc3800f93f52bdd1293f63a2eb34e7f9/src/app/routes/utils/fetchPageData/index.js#L67

I think in Test/Live we default to that `PRIMARY_DATA_TIMEOUT`

Reply:
Oops I was not aware of this. Are we wanting the timeout to be greater (i.e. 60000) then we're using the BFF locally only? And otherwise, let the primary timeout take effect?

Reply:
Yea I think thats what we want. Only locally have the timeout be quite long to give time to let the BFF modules install.

Reply:
I've just made the timeout 60000 if fetching local i.e. fixture data (which is probably overkill) or if running local BFF